docs/game/unreal/getting-started/02-sending-transactions.md

```diff
@@ -9,9 +9,9 @@ This page lists the functionality you can use in your game.
 
 :::tip Write vs. Read methods
 
-* The *Write* method requests — those that change the current state (example: minting NFTs) — incur gas fees to cover smart contract operations. Those requests issue tickets that you need to either approve or reject via MetaMask.
+* The *Write* method requests — those that change the current state (example: minting NFTs) — incur gas fees to cover smart contract operations. Those requests issue tickets you need to approve or reject in MetaMask.
 
-* The *Read* method requests — those that don't change but just show the current state (example: retrieving a balance info) — do not incur gas fees. Therefore, no tickets to be issued or approval needed.  
+* The *Read* method requests — those that don't change but just show the current state (example: retrieving balance info) — do not incur gas fees. Therefore, no tickets to be issued or approval needed.  
```

Comment:
*Read* requests don't change but only show the current state of the blockchain, and don't incur any gas fees.